scene/gui/range.cpp

```diff
@@ -100,6 +100,7 @@ void Range::set_value(double p_val) {
 	shared->emit_value_changed();
 }
 void Range::set_min(double p_min) {
+	ERR_FAIL_COND_MSG(p_min >= shared->max, "Range cannot have min value higher or equal to its max value.");
```

Comment:
This wasn't a very good idea as it puts too strict requirements on how to set min and max values. For example, since the default min and max are 0 and 100, this triggers an error:
```
set_min(256)
set_max(16384)
```
Since `min` will be higher than `max` temporarily. It can be worked around by setting max first, but it's not really intuitive. I'll relax the requirement as it's only a problem in `get_as_ratio`, which already has a check.

Reply:
Fixed with 966c68bad.